packages/amplication-git-pull-request-service/src/diff/build-path-factory.ts

```diff
@@ -1,20 +1,15 @@
 import { Injectable } from "@nestjs/common";
 import { ConfigService } from "@nestjs/config";
 import { join, normalize } from "path";
-import { DEFAULT_BUILDS_FOLDER } from "../constants";
 import { Env } from "../env";
 
 @Injectable()
 export class BuildPathFactory {
   private readonly buildsFolder: string;
   constructor(private readonly configService: ConfigService) {
-    // absolute path to the builds folder
-    const envFilePath = this.configService.get<string>(
-      Env.BUILD_ARTIFACTS_BASE_FOLDER
+    this.buildsFolder = normalize(
+      this.configService.get<string>(Env.BUILD_ARTIFACTS_BASE_FOLDER)
     );
-    this.buildsFolder = envFilePath
-      ? normalize(envFilePath)
-      : DEFAULT_BUILDS_FOLDER;
   }
```

Comment:
By removing the old way of calculating the path you write files directly to the root of the file system `/.amplication/build-artifacts` instead of `/user/.../amplication/.amplication/build-artifacts` this is problematic because we don't want to write files out side of the amplication repository